app/setup.py

```diff
@@ -439,7 +450,6 @@ def setup_babel(application):
     application.babel = Babel(application)
     application.jinja_env.add_extension("jinja2.ext.i18n")
 
-    @application.babel.localeselector
     def get_locale():  # pylint: disable=unused-variable
```

Comment:
This no longer needs to be a method inside the `create_app`

Reply:
`get_locale()` is removed from the create app object